etcdctl/ctlv3/command/lock_command.go

```diff
@@ -28,13 +28,19 @@ import (
 	"golang.org/x/net/context"
 )
 
+var (
+	sessionTTL        int
+	defaultSessionTTL = 10
+)
+
 // NewLockCommand returns the cobra command for "lock".
 func NewLockCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "lock <lockname> [exec-command arg1 arg2 ...]",
 		Short: "Acquires a named lock",
 		Run:   lockCommandFunc,
 	}
+	c.Flags().IntVarP(&sessionTTL, "session-ttl", "t", defaultSessionTTL, "timeout for session")
```

Comment:
modified to --ttl
here is the test:
$ ETCDCTL_API=3 ./etcdctl lock --help
NAME:
        lock - Acquires a named lock

USAGE:
        etcdctl lock <lockname> [exec-command arg1 arg2 ...]

OPTIONS:
      --ttl=10  timeout for session